### ISSUE TITLE
PKG: Create one or more Tasks with dynamic variable values in query

### ISSUE THREAD

**Issue by nisley** [influxdata/influxdb#17081]
My use case -
Create a template that encapsulates https://www.influxdata.com/blog/using-serverless-flux-to-monitor-website-response-times/
When a user installs the template via PKG, PKG sees that input is required and prompts the user for input along the lines of a simple URL, or maybe a key/value list, or maybe a file with those key/values.

For every URL, PKG will create a separate Task and replace the URL in the script with the user input.

**Comment by nisley** [comment]
I started on a new serverless solution to pull Geo data.  The barrier to a full template is my inability to Parameterize/ENVVARIABLE the orgID or orgName in to.()

https://v2.docs.influxdata.com/v2.0/reference/flux/stdlib/built-in/outputs/to/

**Comment by jademcgough** [comment]
This is blocked by https://github.com/influxdata/influxdb/issues/17430
